package.json

```diff
@@ -15,7 +15,12 @@
     "lit-element": "^3.2.2",
     "lit-html": "^2.6.1",
     "@kb-dk/kb-footer": "1.3.0",

```

Comment:
Lets go with the newest version of the footer:)

Reply:
yes